doc/user/content/cli.md

```diff
@@ -380,8 +380,7 @@ We record the following data:
 * System architecture (ARM or x86_64)
 * Operating system type (Linux or macOS)
 
-We use this data to guide our product roadmap. Unless you are using [Materialize
-Cloud](/cloud/), we do not and cannot correlate this
+We use this data to guide our product roadmap, but do not and cannot correlate this
```

Comment:
I think so, too, but is it important to preserve in `lts-docs` if Materialize Cloud doesn't technically exist?

Reply:
We still have a few customers using the legacy Materialize Cloud!

Reply:
Since we're ripping off the band-aid on everything else and keeping close contact with these few users, 🤷‍♀️. I get your concern, though, and it's also a minor mention, so I'll work it back in.

Reply:
> We use this data to guide our product roadmap. Unless you are using the legacy Materialize Cloud product, we do not and cannot correlate this data to your identity.

Reply:
Yeah I just get nervous about an absolute statement like “we cannot correlate this to your identity” when we can, even if it’s just two users. This is great, thank you!